src/main/java/Duke.java

```diff
@@ -1,10 +1,362 @@
+import duke.exception.*;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import java.util.ArrayList;
+import java.util.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static void main(String[] args) throws AssignException, TypingException, FormatException, EventStringException, DeadlineStringException, TodoStringException, IOException {
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
+        int MAX_TASK = 100;
+        String greetings = " Hello! I'm Duke\n" + " What can I do for you?\n";
+        System.out.println(greetings);
+        //Task[] Tasks = new Task[MAX_TASK];
+        ArrayList<Task> Tasks = new ArrayList<>(MAX_TASK);
+        String file = "data/tasks.txt";
+        try {
+            File dukeFile = new File(file);
+            File directory = dukeFile.getParentFile();
+            if(!directory.exists()){
+                directory.mkdir();
+            }
+            if(!dukeFile.exists()){
```

Comment:
Coding standard violation: add spacebar after 'if' statement.

Additionally, consider abstracting this and the above into an external method.